lib/prelude/prelude.elz

```diff
@@ -1,20 +1,13 @@
 // builtin types
-@builtin
```

Comment:
Since we can remove the builtin variant from enum `Type`, we don't need `builtin` tag now!